bench-vortex/src/datasets/tpch_l_comment.rs

```diff
@@ -17,8 +17,6 @@ impl Dataset for TPCHLCommentChunked {
     }
 
     async fn to_vortex_array(&self) -> ArrayRef {
```

Comment:
I removed this since we don't often use this locally and its more confusing to have this implicit build